tests/test_instance.py

```diff
@@ -2,11 +2,14 @@
 from pynio import Instance
 from unittest.mock import MagicMock, patch
 
+
 class TestInstance(unittest.TestCase):
 
-        @patch.object(Instance, '_get_blocks')
         @patch.object(Instance, '_get_services')
-        def test_instance(self, blks, srvs):
+        def test_instance(self, blks):
+            Instance._get_blocks = lambda *args, **kwargs: (None, None)
```

Comment:
I'm not familiar with this. Why would you set the to a new function as opposed to patching _get_objects. You can use `blks.side_effect = (None, None)` to mock return values.